### DISCUSSION TITLE
protectedStrings info

### DISCUSSION THREAD

**Discussion by rayanarfawi** [alex8088/electron-vite#123] [Q&A] [answered]
So to protect a Secure key  in the .env file  MAIN_VITE_SOME_KEY=secretHash
Should i do  
`plugins: [bytecodePlugin({ protectedStrings: ['secretHash'] })]` or `plugins: [bytecodePlugin({ protectedStrings: [Proccess.env.MAIN_VITE_SOME_KEY] })]`

**Comment by alex8088** [comment, marked as answer]
`bytecodePlugin({ protectedStrings: ['secretHash'] })` is ok, the config file will not be packed into the app.

**Reply by rayanarfawi** [reply]
So if i am using in the main js db.password = 'secret'
i should config bytecodePlugin({ protectedStrings: ['secret'] }) and then it will encrypt it in the build.

**Reply by alex8088** [reply]
yes
